dipy/denoise/tests/test_noise_estimate.py

```diff
@@ -29,6 +29,20 @@ def test_piesno():
     sigma = piesno(test_piesno_data, N=8, alpha=0.01, l=1, eps=1e-10, return_mask=False)
     assert_almost_equal(sigma, 0.010749458025559)
 
+    noise1 = (np.random.randn(100, 100, 100) * 50) + 10
+    noise2 = (np.random.randn(100, 100, 100) * 50) + 10
+    rician_noise = np.sqrt(noise1**2 + noise2**2)
+    sigma, mask = piesno(rician_noise, N=1, alpha=0.01, l=1, eps=1e-10, return_mask=True)
+
+    # less than 3% of error?
+    assert_almost_equal(np.abs(sigma - 50) / sigma < 0.03, True)
```

Comment:
they should still work, this test is for the 4D convenience function, the 3D function itself behaves in the same way.